server/app.js

```diff
@@ -33,6 +32,13 @@ const { json, urlencoded } = express;
 
 var app = express();
 
+app.use(
+  cors({
+    origin: "http://localhost:3000", // React App connects from adress
```

Comment:
We shouldn't hardcode dev endpoints

Reply:
This can be an environment variable, or we could use `constants.js` as our environment config